packages/core/src/axis-chart.ts

```diff
@@ -56,6 +56,9 @@ export class AxisChart extends Chart {
 			mainXAxisPosition === AxisPositions.BOTTOM &&
 			mainXScaleType === ScaleTypes.TIME;
 
+		// @todo - should check if zoom bar in all axes are locked
+		const isZoomBarLocked = this.services.zoom.isZoomBarLocked("top");
```

Comment:
`AxisPositions.TOP`

Reply:
Updated in https://github.com/carbon-design-system/carbon-charts/pull/776/commits/f9d15696608945a99fa5b1fb7255dee7bf2ba634.